### PR TITLE
Yr  admin lists calendar view

### PR DESCRIPTION
**Pull request by disel20**
Entity management (by admin). List Management. Calendar view type activation (existing entity)

ticket in trello:
https://trello.com/c/Uh7CmuRu/765-entity-management-by-admin-list-management-calendar-view-type-activation-existing-entity

### CODE REVIEW

File: src/test/java/test/admin/AdminEntityManagementCalendarTest.java [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+
+@Profile(profile = ProfileType.MARKETPLACE)
+@Run(run = RunType.Multiple)
+
+public class AdminEntityManagementCalendarTest extends BaseTest {
+
+    private static final String NEW_ENTITY = TestUtils.randomString(10);
+    private static final String NAME_START_DATE = "Start Date";
+    private static final String NAME_END_DATE = "End Date";
+    private static final String DATE = new SimpleDateFormat("dd/MM/yyyy").format(new Date());
```
Comment (oooleks) [inline src/test/java/test/admin/AdminEntityManagementCalendarTest.java:30]: use currentDate() from TestUtils to get date

File: src/test/java/test/admin/AdminEntityManagementCalendarTest.java [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+
+    @Test
+    public void adminEntityManagementTest() throws ParseException {
+        String[] newCredentials = ProjectUtils.Portal.createInstance(getDriver());
+        String newInstanceLink = newCredentials[0];
+        String newInstancePassword = newCredentials[1];
+
+        LoginPage loginPage = new LoginPage(getDriver())
+                .goToLoginPage(newInstanceLink);
+        loginPage.login("admin", newInstancePassword);
```
Comment (oooleks) [inline src/test/java/test/admin/AdminEntityManagementCalendarTest.java:40]: Try new `ProjectUtils.Portal.createInstanceAndLogin(getDriver());` method instead

File: src/test/java/test/admin/AdminEntityManagementCalendarTest.java [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+import runner.type.Run;
+import runner.type.RunType;
+import utils.ProjectUtils;
+import utils.TestUtils;
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
+@Profile(profile = ProfileType.MARKETPLACE)
+@Run(run = RunType.Multiple)
```
Comment (oooleks) [inline src/test/java/test/admin/AdminEntityManagementCalendarTest.java:23]: No `Multiple `needed if only one test in class.

**Review by oooleks — APPROVED** [review]

**Review by MaxBurdin — APPROVED** [review]

**Review by guram21 — APPROVED** [review]

**Review by GalinaRuban — APPROVED** [review]

File: src/test/java/test/admin/AdminEntityManagementCalendarTest.java (hunk 1 above)
Comment (disel20) [reply src/test/java/test/admin/AdminEntityManagementCalendarTest.java:30]: Done, will update on new PR

File: src/test/java/test/admin/AdminEntityManagementCalendarTest.java (hunk 2 above)
Comment (disel20) [reply src/test/java/test/admin/AdminEntityManagementCalendarTest.java:40]: Done, will update on new PR

File: src/test/java/test/admin/AdminEntityManagementCalendarTest.java (hunk 3 above)
Comment (disel20) [reply src/test/java/test/admin/AdminEntityManagementCalendarTest.java:23]: Thank you for your advices! Will be Done!

**Pull request merged**